### DISCUSSION TITLE
VINDRIKTNING on ESP32 - should it be there by default?

### DISCUSSION THREAD

**Discussion by Saur0o0n** [arendst/Tasmota#15154] [Q&A] [answered]
### PROBLEM DESCRIPTION
I'm not sure if VINDRIKTNING sensor should be available by default in esp32 builds - description says "most drivers including additional sensors", so my understanding is - it should. But I can't find it on the list.
Greping binary file, shows it's somehow defined there.

### REQUESTED INFORMATION

- [x] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)

- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [ ] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x] Device used (e.g., Sonoff Basic): ESP32 D1
- [x] Tasmota binary firmware version number used: 11
  - [x] Pre-compiled
  - [ ] Self-compiled
- [x] Flashing tools used: esptool.py
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:
22:20:40.046 RSL: RESULT = {"NAME":"ESP32-DevKit","GPIO":[1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,0,1,1,1,0,1,1,1,0,0,0,0,1,1,1,1,1,0,0,1],"FLAG":0,"BASE":1}
22:20:40.266 RSL: RESULT = {"Module":{"1":"ESP32-DevKit"}}
22:20:40.529 RSL: RESULT = {"GPIO0":{"0":"None"},"GPIO1":{"0":"None"},"GPIO2":{"0":"None"},"GPIO3":{"0":"None"},"GPIO4":{"0":"None"},"GPIO5":{"0":"None"},"GPIO6":{"0":"None"},"GPIO7":{"0":"None"},"GPIO8":{"0":"None"},"GPIO9":{"0":"None"},"GPIO10":{"0":"None"},"GPIO11":{"0":"None"},"GPIO12":{"0":"None"},"GPIO13":{"0":"None"},"GPIO14":{"0":"None"},"GPIO15":{"0":"None"},"GPIO16":{"0":"None"},"GPIO17":{"0":"None"},"GPIO18":{"0":"None"},"GPIO19":{"0":"None"},"GPIO20":{"0":"None"},"GPIO21":{"0":"None"},"GPIO22":{"0":"None"},"GPIO23":{"0":"None"},"GPIO24":{"0":"None"},"GPIO25":{"0":"None"},"GPIO26":{"0":"None"},"GPIO27":{"0":"None"},"GPIO32":{"0":"None"},"GPIO33":{"0":"None"},"GPIO34":{"0":"None"},"GPIO35":{"0":"None"},"GPIO36":{"0":"None"},"GPIO37":{"0":"None"},"GPIO38":{"0":"None"},"GPIO39":{"0":"None"}}

```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:


```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
Flash tasmota32.bin to any device and look for "VINDRIKTNING" sensor on the list.

### EXPECTED BEHAVIOUR
Find it on the list

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by barbudor** [comment, marked as answer]
No, not included in _any_ binaries whether ESP8266 or ESP32
You need to compile your own build

**Comment by Saur0o0n** [comment]
Ok, It looks like I was mislead a bit. The "recomended" by Tasmota [page on vindriktning](https://blakadder.com/vindriktning-tasmota/), says:

"Or, download an unofficial build [tasmota-allsensors](https://github.com/tasmota/install/raw/main/firmware/unofficial/tasmota-allsensors.bin) which supports almost all sensors, Vindriktning included."

And somehow I've skipped "unofficial" part - since link is to official tasmota repo.

**Reply by barbudor** [reply]
Oh yes indeed there these builds
They are not on the standard Tasmota server but you can get them from Github on the Tasmota account, not on Theo's account
In that case, yes this one should have it

Sorry I often forget about them

**Reply by sfromis** [reply]
Note that what may or may not be included in those "unofficial" builds can be less stable, as an important purpose for some of the binaries is to catch build errors not affecting the "official" builds. The feature mixes are designed for ESP8266 (and often reused for ESP32), meaning that "really all sensors" is not possible due to size restrictions.

**Reply by Saur0o0n** [reply]
I can only confirm that tasmota32-allsensors.bin from unofficial build does have "VINDRIKTNING" sensor.
